traffic_ops/ort/traffic_ops_ort.pl

```diff
@@ -1847,7 +1855,12 @@ sub get_cfg_file_list {
 	my $cdn_name;
 	my $uri = "/api/1.4/servers/$host_name/configfiles/ats";
 
+	# Telling atstccfg to use-cache=false will cause it to delete the cache directory
+	# Which is what we want: when ORT starts to run, delete the cache. We only want to use the atstccfg cache within the same ORT run, not across different runs.
```

Comment:
So, this only works because this is the first request of the run. If someone adds a call before this at some point, they could easily wind up introducing a very subtle race condition. It would be nice to see this a bit more robust? Maybe a global clear-cache flag that is checked and set by lwp_get itself that would run on first call regardless of what that was?

Reply:
Not entirely sure I agree adding a global variable increases the overall robustness. But, I changed it.